cvat-ui/src/components/models-page/model-provider-icon.tsx

```diff
@@ -3,27 +3,24 @@
 // SPDX-License-Identifier: MIT
 
 import React from 'react';
-import { ModelProvider } from 'cvat-core-wrapper';
+import { usePlugins } from 'utils/hooks';
```

Comment:
The component itself better to be moved to a plugin
Or icon should be added for system models

Reply:
This component wasn't used(already moved to plugin), deleted it.